code/datums/outfit/civilian.dm

```diff
@@ -119,7 +119,6 @@
 	..()
 	H.put_in_hands(new /obj/item/weapon/storage/bag/plasticbag(H))
 	H.dna.SetSEState(SOBERBLOCK,1)
-	H.mutations += M_SOBER
```

Comment:
Why ?

Reply:
`H.dna.SetSEState(SOBERBLOCK,1)` already adds M_SOBER to mutations, this causes the bartender to have double Sober mutation and ryetalin will only remove on, which means that bartenders could never get drunk all those years. It's documented in the changelog.

Reply:
Gumshoe had the same issue with both Noir and Sober as was hinted by #26007 so I had a hunch that Bartender might have the same issue, and sure enough it did.

Reply:
Double sober bartender sounds fun though.

Reply:
How so?